ubo-common/src/main/resources/config/ubo-common/mycore.properties

```diff
@@ -535,6 +535,7 @@ MCR.URIResolver.ModuleResolver.ires=org.mycore.ubo.picker.IdentityResolver
 MCR.ContentTransformer.localpidsearch.Stylesheet=xsl/localpidsearch.xsl,%UBO.DefaultHTMLLayoutSteps%
 MCR.IdentityPicker.strategy=org.mycore.ubo.local.Local
 #MCR.IdentityPicker.strategy=org.mycore.ubo.ldap.picker.LDAP
+#MCR.IdentityPicker.strategy=org.mycore.ubo.local.LDAPWithLocal
```

Comment:
have to be
`#MCR.IdentityPicker.strategy=org.mycore.ubo.ldap.picker.LDAPWithLocal`

Reply:
hoppla 🙈